.github/workflows/scripts/countline.py

```
@@ -0,0 +1,296 @@
+    your pull requests.
+
+"""
+
+# Standard imports
+import os
+import sys
+import argparse
+from collections import namedtuple
+
+
+def _valid_filename(filepath):
```

Ruff (0.14.3)

Missing return type annotation for private function `_valid_filename`

(ANN202)